datagrid/datagrid-service-custom-xml.yaml

```diff
@@ -142,10 +142,11 @@ objects:
             timeoutSeconds: 10
           resources:
             limits:
-              memory: ${TOTAL_CONTAINER_MEM}Mi
+              cpu: ${TOTAL_CONTAINER_CPU}
```

Comment:
Had to add cpus here because RHPDS was throwing an error when trying to create the pod

![Screenshot from 2019-05-28 16-20-36](https://user-images.githubusercontent.com/1448375/58509639-065dc780-8165-11e9-9c75-6354528a5135.png)